chirp/src/db/search.rs

```diff
@@ -45,9 +53,9 @@ pub fn db_search(user_query: &String) -> Result<SearchResultsDto, Box<dyn Error>
         collections: collections,
         sources: sources_dtos,
         contents: contents_all_dtos,
-        contents_match_titles: contents_match_titles_dtos,
-        contents_match_bodies: contents_of_bodies_dtos,
-        body_content_ids: body_content_ids,
+        contents_match_titles: vec![],
```

Comment:
This results DTO no longer makes sense now that only a couple of the properties are actually used.